portal/cmd/cmd.go

```diff
@@ -93,8 +94,10 @@ func Execute() {
 
 	// Define global flags relevant to all subcommands and bind them to
 	// the viper configuration chaincode setttings:
+	cfg := &oracle.Config{}
+	cfg.SetDefaults()
 
-	defaultConfig := oracle.DefaultConfig()
+	defaultConfig := cfg
```

Comment:
Will this create issues for separate configs that aren't intended for the oracle package?

Reply:
I've added a wrapper struct now that can contain new vars nore relevant to oracle pkg.